Justfile

```diff
@@ -55,6 +56,7 @@ nextest-unit TYPE *FLAGS:
 # cargo integration tests, TYPE is "stable" or "nightly"
 [linux]
 nextest-integration TYPE *FLAGS:
+    RUSTFLAGS="-D warnings" \
```

Comment:
I actually thought about that too, but then decided against it, because we’ll probably some day want to move crates publishing and fuzzer building into the justfile itself too — so keeping `-D warnings` opt-in sounds like the more future-proof way to do things to me.